### ISSUE TITLE
Accuracy is 0

### ISSUE THREAD

**Issue by Jinyyyy** [yihui-he/channel-pruning#92]
Hi,
When I download the pruned model and run the test on ILSVRC2012 test set, the accuracy is all zero. Does anyone encounter this problem?

This is when I run caffe test -model=channel_pruning.prototxt -weights=channel_pruning.caffemodel -iterations=5000 -gpu=0 for the vgg16 pruning only model:

I0515 11:50:54.617666 25212 caffe.cpp:313] Batch 4983, accuracy@1 = 0
I0515 11:50:54.617697 25212 caffe.cpp:313] Batch 4983, accuracy@5 = 0
I0515 11:50:54.617707 25212 caffe.cpp:313] Batch 4983, loss = 18.8044
I0515 11:50:54.765745 25212 caffe.cpp:313] Batch 4984, accuracy@1 = 0
I0515 11:50:54.765774 25212 caffe.cpp:313] Batch 4984, accuracy@5 = 0
I0515 11:50:54.765784 25212 caffe.cpp:313] Batch 4984, loss = 20.8994
I0515 11:50:54.922375 25212 caffe.cpp:313] Batch 4985, accuracy@1 = 0
I0515 11:50:54.922405 25212 caffe.cpp:313] Batch 4985, accuracy@5 = 0
I0515 11:50:54.922415 25212 caffe.cpp:313] Batch 4985, loss = 28.163
I0515 11:50:55.070338 25212 caffe.cpp:313] Batch 4986, accuracy@1 = 0
I0515 11:50:55.070365 25212 caffe.cpp:313] Batch 4986, accuracy@5 = 0
I0515 11:50:55.070379 25212 caffe.cpp:313] Batch 4986, loss = 17.872
I0515 11:50:55.207842 25212 caffe.cpp:313] Batch 4987, accuracy@1 = 0
I0515 11:50:55.207870 25212 caffe.cpp:313] Batch 4987, accuracy@5 = 0
I0515 11:50:55.207880 25212 caffe.cpp:313] Batch 4987, loss = 17.5856
I0515 11:50:56.942559 25212 caffe.cpp:313] Batch 4988, accuracy@1 = 0
I0515 11:50:56.942596 25212 caffe.cpp:313] Batch 4988, accuracy@5 = 0
I0515 11:50:56.942608 25212 caffe.cpp:313] Batch 4988, loss = 20.6959
I0515 11:50:57.140774 25212 caffe.cpp:313] Batch 4989, accuracy@1 = 0
I0515 11:50:57.140813 25212 caffe.cpp:313] Batch 4989, accuracy@5 = 0
I0515 11:50:57.140823 25212 caffe.cpp:313] Batch 4989, loss = 24.7341
I0515 11:50:57.320144 25212 caffe.cpp:313] Batch 4990, accuracy@1 = 0
I0515 11:50:57.320179 25212 caffe.cpp:313] Batch 4990, accuracy@5 = 0
I0515 11:50:57.320191 25212 caffe.cpp:313] Batch 4990, loss = 18.7019
I0515 11:50:57.464740 25212 caffe.cpp:313] Batch 4991, accuracy@1 = 0
I0515 11:50:57.464771 25212 caffe.cpp:313] Batch 4991, accuracy@5 = 0
I0515 11:50:57.464781 25212 caffe.cpp:313] Batch 4991, loss = 23.8897
